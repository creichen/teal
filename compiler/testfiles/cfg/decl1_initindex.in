fun f(a) = {
  var x := a[0];
}

