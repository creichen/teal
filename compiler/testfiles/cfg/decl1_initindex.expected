Entry#0 @ 1 :
  -> Access#0
VarDecl#0 @ 1 :
VarDecl#1 @ 2 :
  -> Exit#0
Access#0 @ 2 :
  -> IntConstant#0
IndexExpr#0 @ 2 :
  -> VarDecl#1
IntConstant#0 @ 2 :
  -> IndexExpr#0
Exit#0 @ 3 :
